chain/client/src/stateless_validation/state_witness_producer.rs

```diff
@@ -72,6 +86,16 @@ impl Client {
             );
         }
 
+        #[cfg(feature = "contract_distribution")]
```

Comment:
please prefer the runtime check `if cfg!(feature = "contract_distribution")`, the compile check is annoying since it usually causes a bunch of warnings due to unused variables or imports

Reply:
Removed conditional compilation, I will try to use protocol version check as much as possible.